tests/ops/qubit/test_matrix_ops.py

```diff
@@ -269,17 +269,6 @@ def test_qubit_unitary_decomposition(self, U, expected_gate, expected_params):
         assert isinstance(decomp2[0], expected_gate)
         assert np.allclose(decomp2[0].parameters, expected_params, atol=1e-7)
 
-    def test_error_qubit_unitary_decomposition_broadcasted(self):
-        """Tests that broadcasted QubitUnitary decompositions are not supported."""
-        U = np.array(
-            [[0.98877108 + 0.0j, 0.0 - 0.14943813j], [0.0 - 0.14943813j, 0.98877108 + 0.0j]]
-        )
-        U = np.tensordot([1j, -1.0, (1 + 1j) / np.sqrt(2)], U, axes=0)
```

Comment:
Since we're still raising this error for the two qubit case, we should just modify this test instead of delete it.  We need to make sure the error is still raised in the two-qubit case.

We can easily generate a two-qubit broadcasted unitary matrix doing:
```
U = qml.IsingYY.compute_matrix(np.array([1.2, 2.3, 3.4]))
```